cubic-server/configuration/ConfigHandler.cpp

```diff
@@ -11,6 +13,11 @@ namespace Configuration
 
     void ConfigHandler::parse(const std::string &path) {
         YAML::Node config;
+        if (!std::filesystem::exists(path)) {
+            std::ofstream configFile(path);
+            configFile << "network:\n  ip: 0.0.0.0\n  port: 25565\n\ngeneral:\n  max_players: 20\n  motd: A Cubic Server" << std::endl;
```

Comment:
Please put a constexpr variable in the file to be able to change it afterwards.
And also make it a multiline to make editing and reading easier.

Reply:
I pushed a fix, it it what I was supposed to do ? I am not really sure how to use constexpr